dataland-frontend/src/components/general/InfoCard.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <Card class="bg-white d-infocard d-card mr-2">
+    <template #title></template>
+    <template #content>
+      <div class="text-center">
+        <slot> </slot>
+      </div>
+    </template>
+  </Card>
+</template>
+
+<script lang="ts">
+import Card from "primevue/card";
```

Comment:
Dieser untere Teil hier ist ein Duplikat von EuTaxoInfoCard. Kann man das umschreiben damit es kein Duplikat gibt?

Reply:
Meinst du die EuTaxoInfoCard.vue mit der InfoCard.vue zu einer Komponente machen?
Dann müsste man im HTML part der InfoCard.vue eine flag einführen, durch deren Aktivierung das ganze EuTaxo-Zeug zusätzlich gerendert wird.
Wäre theoretisch möglich.  Evtl. ist das aber nicht unbedingt weniger Code. Ich könnte es mal versuchen, was meinst du?

Reply:
Die Recherche hat nichts ergeben, jedoch konnte ich das "options" Objekt, das als Input für die ```defineComponent()``` Funktion dient, zentralisieren.
Schau dir dazu einfach den letzten commit an.

Reply:
Außerdem bitte auch den letzten commit angucken, in dem das Problem mit dem DatalandFooter gelöst wurde.
Achtung: Ich habe versehentlich eine veraltete commit message verwendet.  Diese bitte ignorieren.